<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>632</width>
    <height>455</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>632</width>
    <height>451</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qounter</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
    <property name="spacing">
     <number>6</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="title">
      <property name="font">
       <font>
        <family>Roboto Mono</family>
        <pointsize>40</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Days counter</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="StartDate">
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Start Date (Arrival)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="StartTodayPushButton">
            <property name="text">
             <string>Today</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="StartDateEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCalendarWidget" name="StartCalendar">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="EndLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>End Date (Departure)</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="EndTodayPushButton">
            <property name="text">
             <string>Today</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="EndateEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
              <horstretch>1</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>20</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QCalendarWidget" name="EndCalendar">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>300</width>
            <height>200</height>
           </size>
          </property>
          <property name="verticalHeaderFormat">
           <enum>QCalendarWidget::NoVerticalHeader</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <family>Roboto Mono</family>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Days from today:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="daysFromNow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="maximum">
         <number>99999</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="result">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="textFormat">
       <enum>Qt::RichText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>EndCalendar</sender>
   <signal>clicked(QDate)</signal>
   <receiver>EndateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartCalendar</sender>
   <signal>clicked(QDate)</signal>
   <receiver>StartDateEdit</receiver>
   <slot>setDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>123</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>StartDateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>StartCalendar</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>152</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>152</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EndateEdit</sender>
   <signal>dateChanged(QDate)</signal>
   <receiver>EndCalendar</receiver>
   <slot>setSelectedDate(QDate)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>251</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
